app/src/main/java/com/duckduckgo/app/pixels/remoteconfig/AndroidBrowserConfigFeature.kt

```diff
@@ -179,14 +179,6 @@ interface AndroidBrowserConfigFeature {
     @Toggle.DefaultValue(DefaultFeatureValue.FALSE)
     fun vpnMenuItem(): Toggle
 
-    /**
-     * @return `true` when the remote config has the global "useUnifiedOmnibarLayout" androidBrowserConfig
-     * sub-feature flag enabled
-     * If the remote feature is not present defaults to `true`
-     */
-    @Toggle.DefaultValue(DefaultFeatureValue.TRUE)
```

Comment:
nit: is this going to affect updates? It worked well for me, but just wanted to raise it in case you haven’t tested that scenario.

Reply:
No, this is not used anymore.